bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -16,6 +16,7 @@
 import java.util.List;
 import java.util.concurrent.ExecutionException;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
This must be used anymore. 
See https://github.com/openhab/openhab-addons/issues/7722